assets/js/blocks/breadcrumbs/block.json

```diff
@@ -0,0 +1,47 @@
+{
+	"name": "woocommerce/breadcrumbs",
+	"version": "1.0.0",
+	"title": "Store Breadcrumbs",
+	"description": "Enable customers to keep track of their location within the store and navigate back to parent pages.",
+	"category": "woocommerce",
+	"keywords": [ "WooCommerce" ],
+	"textdomain": "woo-gutenberg-products-block",
+	"attributes": {
+		"contentJustification": {
+			"type": "string"
+		},
+		"fontSize": {
+			"type": "string",
+			"default": "small"
+		},
+		"align": {
+			"type": "string",
+			"default": "wide"
+		}
+	},
+	"supports": {
+		"align": [ "wide", "full" ],
+		"color": {
+			"background": false,
+			"link": true,
+			"__experimentalDefaultControls": {
+				"text": true,
+				"link": true
+			}
```

Comment:
Do you know how to test it? For me, it behaves the same when `__experimentalDefaultControls` is declared and when it's not. :confused: 

(Tested with WP 6.1 and latest Gutenberg)

Reply:
I'm not 100% sure, tbh. The support was added here to be aligned with what is [planned for the native breadcrumbs block on Gutenberg](https://github.com/WordPress/gutenberg/pull/32500/files): I'm glad to take a closer look and re-evaluate if this makes sense.

Reply:
Interesting... If we don't know what it is doing, I would rather not have it in the codebase. What do you think?

Reply:
Oh, I see you removed it in #8345. :slightly_smiling_face: